### PR TITLE
feat(backend): Update scheduledworkflow image to run with pipelinerun

### PR DESCRIPTION
**Pull request by Tomcli**
**Which issue is resolved by this Pull Request:** 
Resolves #

**Description of your changes:**
Update KFP cronjob CRD to handle pipelinerun.

FAQ: Due to KFP still using the old Bazel build, I'm not able to also include Argo because they have conflicts with the k8s go client version. 

**Environment tested:**

* Python Version (use `python --version`): 3.7
* Tekton Version (use `tkn version`): 1.14
* Kubernetes Version (use `kubectl version`): 1.16
* OS (e.g. from `/etc/os-release`):

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

   PR titles examples:
    * `fix(frontend): fixes empty page. Fixes #1234`
       Use `fix` to indicate that this PR fixes a bug.
    * `feat(backend): configurable service account. Fixes #1234, fixes #1235`
       Use `feat` to indicate that this PR adds a new feature. 
    * `chore: set up changelog generation tools`
       Use `chore` to indicate that this PR makes some changes that users don't need to know.
    * `test: fix CI failure. Part of #1234`
        Use `part of` to indicate that a PR is working on an issue, but shouldn't close the issue when merged.

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?
    
    If yes, use one of the following options:
  
    * **(Recommended.)** Ask the PR approver to add the `cherrypick-approved` label to this PR. The release manager adds this PR to the release branch in a batch update.
    *  After this PR is merged, create a cherry-pick PR to add these changes to the release branch. (For more information about creating a cherry-pick PR, see the [Kubeflow Pipelines release guide](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#option--git-cherry-pick).)

### CODE REVIEW

File: backend/src/crd/controller/scheduledworkflow/util/label.go [hunk 1]
```diff
@@ -15,7 +14,7 @@ func GetRequirementForCompletedWorkflowOrFatal(completed bool) *labels.Requireme
 	if completed == true {
 		operator = selection.Equals
 	}
-	req, err := labels.NewRequirement(common.LabelKeyCompleted, operator,
+	req, err := labels.NewRequirement("Completed", operator,
```
Comment (ckadner) [inline backend/src/crd/controller/scheduledworkflow/util/label.go:17]: why not keep using `common.LabelKeyCompleted` here?

Comment (ckadner) [reply backend/src/crd/controller/scheduledworkflow/util/label.go:17]: ah, I see, this if from the `argo` library. Does the Kubeflow or Kubernetes code use constants or plain strings as well? I see Tekton seems to use plain strings but that leave some room for typos causing bugs down the road

Comment (Tomcli) [reply backend/src/crd/controller/scheduledworkflow/util/label.go:17]: This label supposes to come from Argo, but in Tekton it doesn't have the concept of Argo phases, so I was pointing it to some other label. I think there's another label owned by KFP that behaves the similar way, maybe I can switch to that one so we don't have to depend on a string input.

Comment (Tomcli) [reply backend/src/crd/controller/scheduledworkflow/util/label.go]: I updated the label to `LabelKeyWorkflowPersistedFinalState` which is what KFP use to tag for pipelines that are completed.

**Pull request merged**